fs_mark/fs_mark.py

```diff
@@ -0,0 +1,102 @@
+            ('ia32-generic-qemu', 'writeAvg'): [0, 90000],
+            ('ia32-generic-qemu', 'writeMax'): [0, 90000],
+            ('ia32-generic-qemu', 'closeMin'): [0, 90000],
+            ('ia32-generic-qemu', 'closeAvg'): [0, 90000],
+            ('ia32-generic-qemu', 'closeMax'): [0, 90000],
+            ('armv7a9-zynq7000-zturn', 'creatMin'): [],
+            ('armv7a9-zynq7000-zturn', 'creatAvg'): [],
+            ('armv7a9-zynq7000-zturn', 'creatMax'): [],
+            ('armv7a9-zynq7000-zturn', 'writeMin'): [],
+            ('armv7a9-zynq7000-zturn', 'writeAvg'): [],
+            ('armv7a9-zynq7000-zturn', 'writeMax'): [],
+            ('armv7a9-zynq7000-zturn', 'closeMin'): [],
+            ('armv7a9-zynq7000-zturn', 'closeAvg'): [],
+            ('armv7a9-zynq7000-zturn', 'closeMax'): []
+        }
+
+    while True:
+        idx = dut.expect([
+            NAME,
+            MSG_LINE,
+            FINAL,
+            END
+        ], timeout=50)
+        parsed = dut.match.groupdict()
+
+        if idx == 0:
+            test_name = parsed["name"]
+            test_name = test_name[test_name.find("bin/") + 4:]
+        elif idx == 1:
+            splited_line = parsed["line"].split()
+            f_use = splited_line[0]
+            count = splited_line[1]
+            size = splited_line[2]
+            files_sec = splited_line[3]
+            app_overhead = splited_line[4]
+
+            dict = {}
+            dict['creatMin'] = splited_line[5]
+            dict['creatAvg'] = splited_line[6]
+            dict['creatMax'] = splited_line[7]
+            dict['writeMin'] = splited_line[8]
+            dict['writeAvg'] = splited_line[9]
+            dict['writeMax'] = splited_line[10]
+            dict['closeMin'] = splited_line[17]
+            dict['closeAvg'] = splited_line[18]
+            dict['closeMax'] = splited_line[19]
+
+            test_status = Status.OK
+            result = Result(name=test_name, status=test_status)
+            result.status = Status.OK
+
+            for name, value in dict.items():
+                if not timing_dict[(target, name)][0] <= int(value) <= timing_dict[(target, name)][1]:
+                    test_status = Status.FAIL
+                    result.status = Status.FAIL
+                    result.msg += "\n\t" + name + " exec time - " + value + \
+                                  " out of range [" + str(timing_dict[(target, 'creatMin')][0]) + \
+                                  "-" + str(timing_dict[(target, 'creatMin')][1]) + "]"
+                    result.msg += "\n\tF_Use%: " + str(f_use)
+                    result.msg += "\n\tCount: " + str(count)
+                    result.msg += "\n\tSize: " + str(size)
+                    result.msg += "\n\tFiles/sec: " + str(files_sec)
+                    result.msg += "\n\tApp overhead: " + str(app_overhead) + "\n"
+
+            results.append(result)
+            if test_status == Status.FAIL:
+                break
```

Comment:
Do we need to add constant values for a specific test for each value that failed?
 Also why break is needed in this case ?
```suggestion
            for name, value in dict.items():
                if not timing_dict[(target, name)][0] <= int(value) <= timing_dict[(target, name)][1]:
                    result.status = Status.FAIL
                    result.msg += "\n\t" + name + " exec time - " + value + \
                                  " out of range [" + str(timing_dict[(target, 'creatMin')][0]) + \
                                  "-" + str(timing_dict[(target, 'creatMin')][1]) + "]"

            if result.status == Status.FAIL:
                test_status = Status.FAIL
                result.msg += "\n\tF_Use%: " + str(f_use)
                result.msg += "\n\tCount: " + str(count)
                result.msg += "\n\tSize: " + str(size)
                result.msg += "\n\tFiles/sec: " + str(files_sec)
                result.msg += "\n\tApp overhead: " + str(app_overhead) + "\n"

            results.append(result)
```